CHANGELOG.md

```diff
@@ -6,10 +6,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.2] - 2020-08-18
 ### Fixed
-- Y axis legend - offset is calculated using the longest y axis tick value
+- ReadMe - update scatter & line props usage
+- Utils - update sort function props to be more flexible
 
-## [0.3.0] - 2020-08-05
+### Changed
```

Comment:
one important change here is that the packaged `dist` now has less filesize due to `stories` being moved out of `src`